dynawo/sources/Models/CPP/Controls/Voltage/ModelMinMaxMean/test/TestModelMinMaxMean.cpp

```diff
@@ -0,0 +1,414 @@
+//
+// Copyright (c) 2015-2021, RTE (http://www.rte-france.com)
```

Comment:
If I'm not mistaken this file should be deleted

Reply:
You are not mistaken!